Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -283,6 +283,19 @@ class BrowserCoordinator: BaseCoordinator, LaunchCoordinatorDelegate, BrowserDel
             self?.didFinishLibrary(from: libraryCoordinator)
         }
     }
+    private func showETPMenu() {
```

Comment:
I have a nit, could it be possible to add a space between functions in this PR? 

so instead of
```
    }
    private func showETPMenu() {
```

we would have 
```
    }

    private func showETPMenu() {
```

If we make this change, then the other functions should be changed too in your PR to have this spacing.

We sadly don't have any swiftlint rules for this, but this is how we normally format code in the project. It helps the code "breath" a little and is easier on the eyes.